_includes/_head.html

```diff
@@ -1,90 +1,74 @@
-{% assign description = description | strip_html | escape | strip %}
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-{% capture title %}{% if page.meta_title %}{{ page.meta_title}}{% elsif page.title %}{{ page.title }}{% else %}{{ site.title | strip_html | strip }}{% endif %}{% endcapture %}
+<link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
 
-{% assign title = title | strip_html | escape | strip_newlines %}
+{% if site.asciidoctor-enabled == true %}
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.0/css/font-awesome.min.css">
+{% endif %}
 
-{% endcapture %}<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>{{ title }}</title>
+<script src="{{ url }}/assets/js/modernizr.min.js"></script>
 
-    <link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
+<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
+<script>
+  WebFont.load({
+    google: {
+      families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
+    }
+  });
+</script>
 
-  {% if site.asciidoctor-enabled == true %}
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.0/css/font-awesome.min.css">
-  {% endif %}
+<noscript>
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
+</noscript>
 
-	<script src="{{ url }}/assets/js/modernizr.min.js"></script>
+<!-- Search Engine Optimization -->
+{% if page.noindex == true or layout.noindex == true %}<meta name="robots" content="noindex">{% endif %}
 
-	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
-	<script>
-		WebFont.load({
-			google: {
-				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
-			}
-		});
-	</script>
+{% seo %}
 
-	<noscript>
-		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
-	</noscript>
+{% comment %} TODO: Refactor when https://github.com/jekyll/jekyll-seo-tag/pull/148 lands in 2.2.0 {% endcomment %}
+{% if site.bing_webmastertools_id %}<meta name="msvalidate.01" content="{{ site.bing_webmastertools_id }}" >{% endif %}
+{% if site.yandex_site_verification %}<meta name="yandex-verification" content="{{ site.yandex_site_verification }}">{% endif %}
+{% if site.alexa_verify_id %}<meta name="alexaVerifyID" content="{{ site.alexa_verify_id }}">{% endif %}
 
+{% comment %} TODO: Refactor image usage for jekyll-seo-tag {% endcomment %}
+{% if page.image.title %}<meta property="og:image" content="{{ site.urlimg }}{{ page.image.title }}">{% endif %}
+{% if page.image.title %}<meta name="twitter:image" content="{{ site.urlimg }}{{ page.image.title }}">{% endif %}
```

Comment:
Future refactoring should do the following:

- Stop using `image.title` and instead use `image` or `image.path` (or `image.facebook` and `image.twitter`) which is what [Jekyll SEO Tag wants](https://github.com/jekyll/jekyll-seo-tag/#customizing-image-output).
- Allow use of both relative URLs as well as fully qualified URLs. This can be cone using the [`relative-src` include](https://github.com/CloudCannon/frisco-jekyll-template/blob/3ad14657a2c77748ab9ce1fc90d969320dc645fb/_includes/relative-src.html).